.rubocop.yml

```diff
@@ -1479,7 +1479,7 @@ Style/MethodCalledOnDoEndBlock:
   Enabled: false
 
 Style/MethodDefParentheses:
-  Enabled: false
+  Enabled: true
```

Comment:
All subsequent changes were made with `bundle exec rubocop -a`

Reply:
If you believe that there are any methods with arguments that should be able to be defined without parentheses, please let me know!